packages/main/src/backend/export/outputVendors/ynab/ynab.test.ts

```diff
@@ -6,6 +6,42 @@ import * as ynab from './ynab';
 import ClearedEnum = SaveTransaction.ClearedEnum;
 
 describe('ynab', () => {
+  describe('isSameTransaction(Payee changed)', () => {
```

Comment:
I suggest to nest this under the describe for isSameTransaction.
So there is a top level describe for all isSameTransaction tests

Reply:
Agreed